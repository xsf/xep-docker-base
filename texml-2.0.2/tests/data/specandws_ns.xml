<TeXML xmlns="http://getfo.sourceforge.net/texml/ns1">
trademark symbol: &#x2122;
windows&#x2122; outside
// Some time ago it was mapped to 'windows\texttrademark  outside'.
// As result, words 'windows' and 'outside' were not divided by a space.
// Now we use '{}' instead of space, so this test is just a reminder.
</TeXML>
