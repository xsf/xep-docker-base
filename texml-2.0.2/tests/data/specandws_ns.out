trademark symbol: \texttrademark{}
windows\texttrademark{} outside
// Some time ago it was mapped to 'windows\textbackslash{}texttrademark  outside'.
// As result, words 'windows' and 'outside' were not divided by a space.
// Now we use '\{\}' instead of space, so this test is just a reminder.
